tyk-stack/values.yaml

```diff
@@ -34,15 +34,16 @@ global:
     password: "123456"
 
   components:
+    # Determines whether the bootstrap jobs should be run
+    # Those jobs verify the presence of the dashboard license and perform various operations to
+    # make apps such as tyk-dashboard, tyk-portal and tyk-operator ready to use from the get go
+    # If this is set to false, only the hook that checks for dashboard license will be run
```

Comment:
This may cause confusion such that the hook responsible for validating dashboard license only runs when `components.bootstrap` is set to false. What do you think about updating this comment